src/hardware_trigger.cpp

```diff
@@ -183,6 +203,49 @@ HardwareTrigger::condition HardwareTrigger::digitalCondition(uint chnIdx)
 	return static_cast<condition>(it - lut_digital_trigg_cond.begin());
 }
 
+
+
+bool HardwareTrigger::hasExternalTriggerOut()
+{
+	auto ret = iio_channel_find_attr(m_logic_channels[1], "out_select");
+
+	return (ret==NULL) ? false : true;
+
```

Comment:
There are some extra lines here, also there is no empty line after the `}` of this method.

Reply:
fixed